sympy/printing/fcode.py

```diff
@@ -712,7 +712,7 @@ def _print_FunctionPrototype(self, fp):
             "{function_head}\n"
             "end function\n"
             "end interface"
-        ).format(function_head=self._head(entity, fp, *args))
```

Comment:
`args` not defined. Should be possible to at least cover the code in a test (although the test basically will be "does not set a variable is undefined")

Reply:
I have confirmed that the old version gives:

...
  File "C:\Users\Oscar\sympy\sympy\printing\fcode.py", line 715, in _print_FunctionPrototype
    ).format(function_head=self._head(entity, fp, *args))

NameError: name 'args' is not defined
...

The new version leads to:

...
  File "C:\Users\Oscar\sympy\sympy\printing\fcode.py", line 479, in _print_Declaration
    raise NotImplementedError("F77 init./parameter statem. req. multiple lines.")

NotImplementedError: F77 init./parameter statem. req. multiple lines.
...

So I guess not much point in writing a test before that is implemented?

Reply:
> writing a test before that is implemented?

There are also tests for raising NotImplementedError (in polys/tests), possibly in expectation of future implementation.

Reply:
Yes, I thought about that as well. Seems to not be the actual purpose of writing tests expecting an exception, but I can for sure add one with relevant comments. Probably should add an issue for multiline printing as well (unless there already is one, will search).